sleyster_a07b/part_2/js/script.js

```diff
@@ -0,0 +1,131 @@
+/*Heather Sleyster
+sleyster_a07bpart2
+Thoendel
+INFO 2134
+04/23/2020
+*/
+
+window.addEventListener('load', () => {
+    const right = document.getElementById('right');
+    const left = document.getElementById('left');
+
+displayLoading('left', 'Loading employee list ...');
+
+const url = 'https://www.mccinfo.net/epsample/employees';
+
+fetch(url)
+    .then (response => response.json() )
+        .then ( (employees => {
+            console.log(employees);
+
+            let para = document.createElement('p');
+            para.innerHTML = 'Select an employee from the list below: ';
+
+            let employeeList = document.createElement('select');
+            employeeList.id = 'employeeList';
+
+            let option = document.createElement('option');
+            option.value = '';
+            option.innerHTML = '--Select an option--';
+            employeeList.appendChild(option);
+
+            for(const employee of employees){
+                option = document.createElement('option');
+                option.value = employee.id;
+                option.innerHTML = `${employee.first_name} ${employee.last_name} (${employee.department.name})`;
+                employeeList.appendChild(option); 
+            }
+            clearContainer('left');
+            left.appendChild(para);
+            left.appendChild(employeeList);
+
+            employeeList.addEventListener('change', onChange);
+        //Begin Step 2
+            employeeList.addEventListener('select', displayEmployeeInfo('right', employeeList.first_name, 
+            employeeList.last_name, employeeList.department.name, employeeList.annual_salary, employeeList.hire_date));       
+        //End Step 2
+        })
+
+        //Begin Step 3
+
+const imageUrl = 'https://mccinfo.net/epsample/images/' + employeeList.last_name + '.jpg';
```

Comment:
employeeList only exists in the scope above - not here. You did, however, define employees (I would have kept it to employee) on line 18. That is really what you need to use to access aspects of this employee. Make sense?